src/services/monaco-editor.service.ts

```diff
@@ -93,10 +102,7 @@ export class MonacoEditorService {
 		return this.bootstrap$.pipe(
 			tap(() => {
 				// Dispose all the current models
-				for (const model of monaco.editor.getModels()) {
-					model.dispose();
-				}
-
+        this.disposeModels();
```

Comment:
Can you fix this one as well? :)

Reply:
Done!

Reply:
Did you forgot to push?

Reply:
Nope I pushed. I think it's fast if you just fix it yourself.